launchlib/launcher.go

```diff
@@ -91,6 +91,22 @@ func CompileCmdFromConfig(staticConfig *StaticLauncherConfig, customConfig *Cust
 	return createCmd(executable, args, env)
 }
 
+func MkDirs(dirs []string) error {
+	isDirMatcher := regexp.MustCompile(`^[A-Za-z0-9]+(/[A-Za-z0-9]+)*$`).MatchString
```

Comment:
Probably fine in this case, but in long-lived applications it's typically better to store the result of `regexp.MustCompile` in a package-private variable so that it's parsed/compiled once